requirements.txt

```diff
@@ -1,5 +1,4 @@
 pytest
 virtualenv
 yapf==0.24.0
-intervaltree
-simplejson
+-e ./
```

Comment:
So I don't think prjuray-tools should require itself?

Reply:
It does not. But these are requirements for the whole prjuray-tools repo. The ones for prjuray module are in setup.py. Having this line here allows prjuray to be installed in the venv for CI tests.

Reply:
I left the line as is because `requirements.txt` does not belong to the `prjuray` package. It is used here by the CI to install and test it.

When someone wants to install `prjuray` only then one will use `setup.py`. But when one wants to use the whole `prjuray-tools` (which is not a python package) then one will invoke `pip install -r` and that will install `prjuray`.